lib/cc/engine/analyzers/engine_config.rb

```diff
@@ -0,0 +1,49 @@
+module CC
+  module Engine
+    module Analyzers
+      class EngineConfig
+        attr_reader :config
+
+        def initialize(hash)
+          normalize(hash)
+        end
+
+        def language(language)
+          config.
+            fetch("languages", {}).
```

Comment:
We should use the `DEFAULT_LANGUAGE`, which is Ruby.

https://github.com/codeclimate/codeclimate-duplication/blob/bmw-engine-config-refactor/lib/cc/engine/duplication.rb#L49-L53